examples/triangle/triangle.zig

```
@@ -169,6 +162,7 @@ pub fn main() !void {
         defer staging_buffer.unmap();
 
         const output = buf[0..output_size];
+        std.fs.cwd().makeDir("examples/output") catch {};
```

I prefer not to suppress all errors here; if this were to fail for some other reason than `PathAlreadyExists`, you'd see an error show up in `write24BitBMP()` instead and it makes tracing the actual source of the error a little more difficult. It seems that `std.fs.Dir` has a `makePath()` function that handles the case where the directory already exists, and returns any other type of error, so maybe you could do `try std.fs.cwd().makePath("examples/output");`